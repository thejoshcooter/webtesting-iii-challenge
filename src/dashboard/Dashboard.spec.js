import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';


import Dashboard from './Dashboard';

describe('<Dashboard />', () => {
    it ('renders display/controls', () => {
        const tree = renderer.create(<Dashboard />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})