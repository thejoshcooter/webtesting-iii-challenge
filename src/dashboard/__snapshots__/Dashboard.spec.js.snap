// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> renders display/controls 1`] = `
Array [
  <div
    className="display panel"
  >
    <div
      className="led green-led"
      data-testid="lockUnlock"
    >
      Unlocked
    </div>
    <div
      className="led green-led"
      data-testid="openClose"
    >
      Open
    </div>
  </div>,
  <div
    className="controls panel"
  >
    <button
      className="toggle-btn"
      data-testid="lockUnlock"
      disabled={true}
      onClick={[Function]}
    >
      Lock Gate
    </button>
    <button
      className="toggle-btn"
      data-testid="openClose"
      disabled={false}
      onClick={[Function]}
    >
      Close Gate
    </button>
  </div>,
]
`;
